src/apps/deskflow-gui/dialogs/AboutDialog.cpp

```diff
@@ -89,5 +89,11 @@ AboutDialog::AboutDialog(QWidget *parent) : QDialog(parent)
 
 void AboutDialog::copyVersionText()
 {
-  QGuiApplication::clipboard()->setText(m_versionString);
+  QString infoString = QStringLiteral("Deskflow: %1 (%2)\n Qt: %3\n System: %4")
```

Comment:
I'm guessing those extra spaces before 'System' and 'Qt' weren't intentional?

```suggestion
  QString infoString = QStringLiteral("Deskflow: %1 (%2)\nQt: %3\nSystem: %4")
```

With the extra spaces it looks like:
```
Deskflow: 1.18.0.66 (171a0b6b)
 Qt: 6.8.1
 System: Arch Linux
Session: KDE (wayland)
```

But should look like:
```
Deskflow: 1.18.0.66 (171a0b6b)
Qt: 6.8.1
System: Arch Linux
Session: KDE (wayland)
``

Reply:
Done